internal/presenters/test_results.gotmpl

```diff
@@ -0,0 +1,22 @@
+[1m[37m[39m[22m
```

Comment:
Question: How do we consider/support different TTY types that have different capabilities?

Reply:
There are two options we could take here:
* strip them out at a later date in the process
* Adopt a library like https://github.com/charmbracelet/lipgloss which handles this automagically for us 💜